demo/gphl/gphl-setup.yml

```diff
@@ -43,8 +43,8 @@ configuration:
   # HardwareRepository directories on the lookup path
 
 software_paths:
-  # Mandatory. Directory with workflow config input, e.g. instrumentation.nml
-  gphl_beamline_config: gphl/config_esrf_id30b
+    # Mandatory. Directory with workflow config input, e.g. instrumentation.nml
+    gphl_beamline_config: gphl/config_esrf_id30b
```

Comment:
@rhfogh I had to disable the GPHL workflows for the moment because the parsing of the YAML files failed. Perhaps you know what the issue is ?

Reply:
Actually it was the parsing by `raumel.load` that failed for some reason. I can have another look at a later stage now when everything is else seems to back on track working. But I can't look at it right away as I have a bunch of other things to finish as well.

Reply:
I had a look and I fixed a indentation error, not sure what the original error actually was but it seems to be loading now.